tests/test_concurrentTransmissions.py

```diff
@@ -1,19 +1,64 @@
 import Wireless
-
+import pytest
 # ============================ fixtures ==============================
+EXPECTEDINOUT = [
+
+    {
+        'in': {
```

Comment:
this is very obscrure as you have a list of PDRs, then this A, B, C thing. Make simpler: explicitly list the PDRs between a handful of nodes. Also, you only test with 2 PDRs alwats